misc/package/Data Files/MWSE/core/lib/dependencyManagement/DependencyManager.lua

```diff
@@ -74,8 +78,53 @@ end
 
 
 ---Check if all dependencies are met
----@return boolean #returns true if all dependencies passed, false if any failed.
+---@return boolean passed #returns true if all dependencies passed, false if any failed.
 function DependencyManager:checkDependencies()
+    if not mwse.getConfig("EnableDependencyChecks") then
+        return true
+    end
+
+    -- Don't check dependencies if a mod't plugin is not active.
+    -- Log if there is metadata.toml pointing to a partially installed/uninstalled mod.
+    local plugin = self.metadata.package.plugin
+    local pluginExists = plugin and util.pluginExists(plugin)
+    local pluginDisabled = pluginExists and not tes3.isModActive(plugin)
+
+    local luaMod = self.metadata.tools and self.metadata.tools.mwse and self.metadata.tools.mwse["lua-mod"]
+    local luaModExists = luaMod and util.luaModExists(luaMod)
+
+    local luaOnly = luaMod and not plugin or false
+    local pluginOnly = plugin and not luaMod or false
+    local luaPlugin = luaMod and plugin and true or false
```

Comment:
Put parentheses around the condition, i.e `(luaMod and not plugin) or false` so its easier to see what these are doing

Reply:
In fact, doing so reveals a bug:

`local luaPlugin = (luaMod and plugin and true) or false`

can never be false